pysat/instruments/sw_f107.py

```diff
@@ -411,8 +416,14 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
         # process
         raw_dict = json.loads(r.text)['noaa_radio_flux']
         data = pds.DataFrame.from_dict(raw_dict['samples'])
-        times = [pysat.datetime.strptime(time, '%Y %m %d')
-                 for time in data.pop('time')]
+        try:
+            # This is the new data format
+            times = [pysat.datetime.strptime(time, '%Y%m%d')
+                     for time in data.pop('time')]
+        except ValueError:
```

Comment:
same as above

Reply:
Taking out the try/except statement should take care of coverall.s